### PR TITLE
Vaccination Tab icon changes and Remove add brother sister link from header 

### PR DESCRIPTION
**Pull request by DivyaniPatel96**
Vaccination Tab icon changes and Remove add brother sister link from header

### CODE REVIEW

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -3,13 +3,13 @@ name: Android Build
 # Controls when the workflow will run
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
```
Comment (Akhror) [inline .github/workflows/android.yml:5]: Update the comment

**Review by Akhror — APPROVED** [review]

File: .github/workflows/android.yml (hunk 1 above)
Comment (Akhror) [reply .github/workflows/android.yml:5]: name: Android Build

on:
  push:
    branches:
      - vaccinationTabIconChange
  workflow_dispatch:

jobs:
  build:
    name: Build and Test
    runs-on: ubuntu-latest
    strategy:
      matrix:
        node-version: [16.14.0]
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Setup Node.js
        uses: actions/setup-node@v2
        with:
          node-version: ${{ matrix.node-version }}

      - name: Cache NDK
        uses: actions/cache@v2
        with:
          path: ${{ env.ANDROID_HOME }}/ndk/21.0.6113669
          key: ${{ runner.os }}-ndk-${{ env.MATRIX }}-${{ hashFiles('**/build.gradle*') }}

      - name: Install NDK
        run: echo "y" | sudo ${ANDROID_HOME}/tools/bin/sdkmanager --install 'ndk;21.0.6113669'

      - name: Install dependencies
        run: npm ci

      - name: Lint
        uses: Krizzu/eslint-check-action@v1.1.0
        with:
          ghToken: ${{ secrets.GITHUB_TOKEN }}
          eslintFiles: "app"
          eslintConfig: ".eslintrc.json"
          eslintExt: "js, jsx, ts, tsx"

      - name: Create environment file
        run: |
          echo "apiUrlDevelop=${{ secrets.APIURLDEVELOP }}" >> .env
          echo "facebookAppDisplayName=${{ secrets.FACEBOOKAPPDISPLAYNAME }}" >> .env
          echo "facebookAppId=${{ secrets.FACEBOOKAPPID }}" >> .env
          echo "facebookClientToken=${{ secrets.FACEBOOKCLIENTTOKEN }}" >> .env
          echo "projectNumber=${{ secrets.PROJECTNUMBER }}" >> .env
          echo "clientIdKey=${{ secrets.CLIENTIDKEY }}" >> .env

      - name: Decode google-services.json
        run: |
          echo "${{ secrets.GOOGLE_SERVICES_JSON }}" | base64 --decode > android/app/src/prodstaging/google-services.json

      - name: Decode keystore and create properties file
        run: |
          echo "$MYAPP_UPLOAD_STORE_FILE" | base64 --decode > app/bebborelease.keystore
          echo "storeFile=bebborelease.keystore" >> key.properties
          echo "keyAlias=$KEYSTORE_KEY_ALIAS" >> key.properties
          echo "storePassword=$KEYSTORE_STORE_PASSWORD" >> key.properties
          echo "keyPassword=$KEYSTORE_KEY_PASSWORD" >> key.properties
        env:
          MYAPP_UPLOAD_STORE_FILE: ${{ secrets.ANDROID_SIGNING_KEY }}
          KEYSTORE_KEY_ALIAS: ${{ secrets.ANDROID_ALIAS }}
          KEYSTORE_STORE_PASSWORD: ${{ secrets.ANDROID_KEY_PASSWORD }}
          KEYSTORE_KEY_PASSWORD: ${{ secrets.ANDROID_KEY_STORE_PASSWORD }}

      - name: Cache Gradle
        uses: actions/cache@v2
        with:
          path: ~/.gradle/caches
          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
          restore-keys: |
            ${{ runner.os }}-gradle-

      - name: Make Gradlew Executable
        run: chmod +x ./gradlew

      - name: Build
        run: |
          cd android
          ./gradlew clean assembleprodstagingRelease --no-daemon

      - name: Upload Artifact
        uses: actions/upload-artifact@v2
        with:
          name: Bebbo_Release.apk
          path: android/app/build/outputs/apk/

Comment (Akhror) [reply .github/workflows/android.yml:5]: Following points have been improved in above version by AI tool:
1. Introduced a matrix strategy to test against different Node.js versions. This allows parallel testing across different environments, speeding up the process.
2. Caching strategies have been refined for NDK and Gradle to ensure cache hits and misses are handled efficiently, speeding up subsequent runs.
3. Steps like linting, creating environment files, and decoding files have been optimized to run concurrently, reducing overall build time.
Please compare the code and consider using it if it looks better.

Comment (BhumikaDatamatics) [reply .github/workflows/android.yml:5]: @Akhror We will share supported node versions for upgraded reactnative version. as build can be created in the support node versions only. 
Please clarify the 2 and 3rd point. we can discuss these separately. 

As of now, We are merging this request to the main branch as we need to start work on the build process.

**Pull request merged**